pubspec.yaml

```diff
@@ -1,11 +1,16 @@
 name: platform
-version: 3.1.0
-description: A pluggable, mockable platform abstraction for Dart.
-homepage: https://github.com/google/platform.dart
+description: A pluggable, mockable platform information abstraction for Dart.
+repository: https://github.com/dart-lang/platform
+issue_tracker: https://github.com/dart-lang/platform/issues
+version: 3.1.4
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ^3.2.0
 
 dev_dependencies:
-  lints: ^1.0.1
+  lints: ^4.0.0
```

Comment:
Use `dart_flutter_team_lints`?
(Not that I prefer them, but they appear to be the law.)

Reply:
Done